tools/hardware/config.py

```diff
@@ -37,11 +37,18 @@ class ARMConfig(Config):
 
     def __init__(self, sel4arch, addrspace_max):
         super().__init__(sel4arch, addrspace_max)
-        self.SUPERSECTION_BITS = 25 if sel4arch == 'arm_hyp' else 24
+        # On AArch32 the kernel requires at least super section alignment for physBase.
+        # Page sizes differ on arm_hyp which is why the alignment changes even though
+        # both arm_hyp and arm are 32-bit.
+        if sel4arch == 'arm_hyp':
+            self.KERNEL_PHYS_ALIGN = 25
+        elif sel4arch == 'arm':
```

Comment:
I think this should be aarch32 because the valid options are: `'aarch32', 'aarch64', 'arm_hyp'`

Reply:
Fixed, thanks.